doc/source/whatsnew/v0.20.0.txt

```diff
@@ -268,6 +268,7 @@ To convert a ``SparseDataFrame`` back to sparse SciPy matrix in COO format, you
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- ``DataFrame.rolling()`` now accepts the parameter ``closed='right'|'left'|'both'|'neither'`` to choose the rolling window endpoint closedness. (:issue:`13965`)
```

Comment:
also add a ``:ref:`` to the new sub-section that you are creating in computation.rst